identity/app/model/EmailSubscriptions.scala

```diff
@@ -97,6 +97,7 @@ object EmailSubscriptions {
       frequency = "Tuesday mornings",
       listId = "3698",
       subscribedTo = subscribedListIds.exists{ x => x == "3698" },
+      signupPage = Some("https://www.theguardian.com/politics/2016/may/31/eu-referendum-morning-briefing-sign-up"),
```

Comment:
I've had a look around in frontend.. and while it seems possible to change the subdomain/route linked to, dependent on the environment (CODE/PROD), it is not currently being done for identity, and may be adding more complexity than value.

I could try something hacky with /common/app/common/configuration.scala
`@if(Configuration.environment.isNonProd){ ...`

This would solve for code, but still doesn't help with linking to the correct path of localhost